src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -420,21 +420,30 @@ private EmailWrapper generateFeedbackSessionEmailBaseForStudents(
         return email;
     }
 
+    private String generateInstructorFragment(String courseId, String courseName) {
+
+        String courseIdentifier = SanitizationHelper.sanitizeForHtml(courseId) + ", "
+                + SanitizationHelper.sanitizeForHtml(courseName);
+
+        return "<p>The email below has been sent to students of course: "
+                + courseIdentifier
+                + ".<br>" + Const.EOL + "<br>" + Const.EOL
+                + "=== Email message as seen by the students ===</p>" + Const.EOL;
```

Comment:
This line wrapping is easier to map to the final output. e.g. the fullstop appears near to `courseIdentifier ` and `<br>` appears at the end of the line.
```java 
return "<p>The email below has been sent to students of course: "
        + courseIdentifier + ".<br>" + Const.EOL 
        + "<br>" + Const.EOL
        + "=== Email message as seen by the students ===</p>" + Const.EOL;
```